src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -457,10 +429,10 @@ protected function prepare_links( $id ) {
 	 *
 	 * @return array List of link relations.
 	 */
-	protected function get_available_actions() {
+	protected function get_available_actions( $post, $request ) {
```

Comment:
Missing `@since` and `@param` decoration for variables.

Reply:
@mukeshpanchal27 Fixed.